src/app/inventory/store/d2-item-factory.ts

```diff
@@ -597,13 +597,30 @@ export function makeItem(
     reportException('Quest', e, { itemHash: item.itemHash });
   }
 
-  // TODO: Phase out "base power"
   if (createdItem.primStat) {
     createdItem.basePower = createdItem.primStat.value;
   }
 
   createdItem.index = createItemIndex(createdItem);
 
+  // Some items have multiple tooltips, but the item.tooltipNotificationIndexes property that
+  // should tell us which to show is missing: https://github.com/Bungie-net/api/issues/1419
+  if (
+    itemDef.tooltipNotifications?.length === 1 &&
+    itemDef.tooltipNotifications[0].displayString.length
```

Comment:
could this work in this case? optional chaining works on array indices as well`itemDef.tooltipNotifications.?[0].displayString.length`

Reply:
ah, hmm unless you're checking for the length to specifically be "`1`"?

Reply:
I am looking for it to be 1.